.changeset/gorgeous-suits-camp.md

```diff
@@ -0,0 +1,7 @@
+---
+'@aws-amplify/backend-data': minor
+'@aws-amplify/plugin-types': minor
+'@aws-amplify/backend': minor
```

Comment:
```suggestion
'@aws-amplify/backend-data': patch
```

1. We're not shipping new apis nor features, just performance optimization
2. Plugin types are no longer affected.
3. We don't need to bump `backend`. package managers will resolve new backend-data when they upgrade deps graph or do new installation.